### PR TITLE
installation: fix invenio_utils imports

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -28,3 +28,5 @@
 -e git+git://github.com/inveniosoftware/invenio-formatter.git#egg=invenio-formatter
 -e git+git://github.com/inveniosoftware/invenio-search.git#egg=invenio-search
 -e git+git://github.com/inveniosoftware/invenio-upgrader.git#egg=invenio-upgrader
+-e git+git://github.com/inveniosoftware/invenio-utils.git#egg=invenio-utils
```
Comment (jirikuncar) [inline requirements-devel.txt]: please keep the file sorted

Comment (hachreak) [inline requirements-devel.txt]: done

File: setup.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
     'invenio-formatter>=0.2.1',
     'invenio-search>=0.1.3',
     'invenio-upgrader>=0.1.0',
+    'invenio-ext>=0.2.1',
```
Comment (jirikuncar) [inline setup.py]: missing `invenio-utils` + sorting

**Comment by jirikuncar** [comment]
@hachreak please rebase and :shipit:

**Comment by hachreak** [comment]
done

**Pull request merged**